requirements.txt

```diff
@@ -3,15 +3,15 @@ cycler==0.10.0
 kiwisolver==1.1.0
 matplotlib==3.1.3
 numpy==1.18.1
-opencv-python==3.3.1
+opencv-python==3.4.2.17
+openpyxl>=2.6.1
```

Comment:
The very first version of pysero used openpyxl to load and create excel spreadsheets.  We are using pandas now instead.

This means we can also remove two files:
array_analyzer/load/xlsx_report.py
array_analyzer/load/xlsx_report_nb.ipynb

Reply:
I'm the only one who worked on those files :-).  I think openpyxl is too cumbersome, and @smguo mentioned in slack that he had difficulty installing it on PC.